packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -298,6 +306,22 @@ struct WindowData {
             self->m_useDirectDebugger = IsDlgButtonChecked(hwnd, IDC_DIRECTDEBUGGER) == BST_CHECKED;
             self->m_breakOnNextLine = IsDlgButtonChecked(hwnd, IDC_BREAKONNEXTLINE) == BST_CHECKED;
 
+            LPWSTR buffer = new WCHAR[BUFFER_LENGTH];
```

Comment:
>new WCHAR [](start = 28, length = 9)

This buffer is leaking memory.
As a rule of thumb we must avoid using a raw 'new' operator call that allocates memory.
While I usually suggest to use std::make_unique in such cases, here it seems that having a buffer on the call stack is much better option:

WCHAR buffer[BUFFER_LENGTH] = {};

Reply:
My bad, thanks for catching this!